nextflow/ProteinFunction/main.nf

```diff
@@ -193,16 +217,32 @@ workflow {
                           name: "translation_mapping.tsv",
                           storeDir: params.outdir,
                           newLine: true) { it.id + "\t" + it.md5 }
-  store_translation_mapping(translation_mapping, translation_mapping_wait)
-  store_assemblies(files.collect(), assemblies_wait)
+
+  if (!params.offline) {
+    store_translation_mapping(translation_mapping)
+    store_assemblies(files.collect(), assemblies_wait)
+  }
 
   // Get unique translations based on MD5 hashes of their sequences
   translated = translated.unique { it.md5 }
 
   // Run protein function prediction
   errors = Channel.of("# failure reasons")
-  if ( params.sift_run_type != "NONE" ) errors = errors.concat(run_sift_pipeline( translated ))
-  if ( params.pph_run_type  != "NONE" ) errors = errors.concat(run_pph2_pipeline( translated ))
+
+  if ( params.sift_run_type != "NONE" ) {
+    errors = errors.concat(run_sift_pipeline( translated, sqlite_db_prep ))
+  } else {
+    sift_run = "done"
```

Comment:
Hi @nakib103 , we may need to define `sift_run` when params.sift_run_type != "NONE"

Reply:
Thanks! it has been fixed.